liam2/alignment.py

```diff
@@ -536,8 +575,12 @@ def __init__(self, score=None, proportions=None,
                                 "arguments should have the same length")
 
         if proportions is None and fname is None:
-            raise Exception("align() needs either an fname or proportions "
-                            "arguments")
+            if method == 'bysorting':
+                raise Exception("align() needs either an fname or proportions "
+                                "arguments")
+            elif method == 'sidewalk':
+                raise Exception("If alignment on sum of score is wanted "
+                                "then use align_abs")

```

Comment:
It took me several reads to understand what this means. Any idea to make this clearer?